plugin/orderfield.go

```diff
@@ -0,0 +1,40 @@
+package plugin
+
+import (
+	"github.com/gogo/protobuf/protoc-gen-gogo/generator"
+)
+
+type OrmOrderField struct{}
+
+func (o *OrmOrderField) getOrderedFieldNames(ormable *OrmableType, message *generator.Descriptor) (fields []string) {
```

Comment:
As I stated in my previous comment, `getOrderedFieldNames` function should only return field names which exist in ORM, so it can be safely used across the plugin. So before appending a field to `fields` you should check if it exists in ORM model.

Reply:
Hi @dkukharau , About it `So before appending a field to fields you should check if it exists in ORM model.` - I checking field ORM in generateOrmable func, line 226. If need checking in my function i will do it. What do you think? :)
```
field, ok := ormable.Fields[fieldName]
if ok {
	p.P(fieldName, ` `, field.Type, p.renderGormTag(field))
}
```

Reply:
Yeah, I see that.
But as I stated in my previous comments:
`what I expect is that getOrderedFieldNames function should only return field names which exist in ORM, so it can be safely used across the plugin.`
I want this function to be used instead of `getSortedFieldNames`, and that function was supposed to return only ormable fields.
Thanks!